src/config/ConfigBuilder.ts

```diff
@@ -493,27 +495,67 @@ export class ConfigBuilder {
         }
     }
 
+    private handleCompactSerializationConfig(compactSerializationConfig: any) {
+        for (const key in compactSerializationConfig) {
+            if (key === 'serializers') {
+                this.handleCompactSerializers(compactSerializationConfig[key]);
+            } else {
+                throw new RangeError(`Unexpected compact serialization config '${key}' is passed to the Hazelcast Client`);
```

Comment:
I added LazyDeserializationCompactTest for ReadResultSet and ReadOnlyLazyList data structures.

Reply:
I am not going to go over all of the warnings because I once did that in the past and there were a lot of uncovered places. I remember I fixed all warnings back then. Can you point me the important ones, because there may be some introduced later on as I continued refactoring, so that I can fix them as well?